tests/unittest/auto_deploy/singlegpu/custom_ops/mla/test_trtllm_mla_op.py

```
@@ -0,0 +1,1514 @@
+    kv_lora_rank,
+    qk_rope_head_dim,
+    dtype,
+    device,
+):
+    """Build TRT-LLM metadata using explicit page assignments.
+
+    Unlike _create_trtllm_paged_metadata (which assigns pages sequentially from
+    scratch), this function takes pre-computed page assignments so that page
+    mappings remain consistent across prefill and decode steps.
+    """
+    kv_lengths = [pos + slen for pos, slen in zip(input_positions, seq_lengths)]
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)